library/src/permutation/device/hiptensor_permutation_scale_instances.hpp

```diff
@@ -45,128 +45,186 @@ namespace ck
                 using F32 = float;
 
                 // clang-format off
-                template <typename AOp,
-                          typename BOp,
-                          typename Scale,
-                          index_t NDims>
+                template <index_t NDims,
+                          typename ElementwiseOp>
                 using device_permute_scale_f16_instances =
                     std::tuple <
-                        DeviceElementwiseImpl<ck::Tuple<F16>, ck::Tuple<F16>, AOp, BOp, Scale, NDims, 1, ck::Sequence<1>, ck::Sequence<1>>,
-                        DeviceElementwiseImpl<ck::Tuple<F16>, ck::Tuple<F16>, AOp, BOp, Scale, NDims, 8, ck::Sequence<8>, ck::Sequence<1>>,
-                        DeviceElementwiseImpl<ck::Tuple<F16>, ck::Tuple<F16>, AOp, BOp, Scale, NDims, 4, ck::Sequence<4>, ck::Sequence<1>>,
-                        DeviceElementwiseImpl<ck::Tuple<F16>, ck::Tuple<F16>, AOp, BOp, Scale, NDims, 2, ck::Sequence<2>, ck::Sequence<1>>
+                        DeviceElementwiseImpl<ck::Tuple<F16>, ck::Tuple<F16>, ElementwiseOp,  NDims, 256, 64,  64,  4, 4, ck::Sequence<1, 0>, ck::Sequence<4>, ck::Sequence<4>>,
```

Comment:
These are defined again in library/src/permutation/device/permutation_scale_f16_instances.hpp